### DISCUSSION TITLE
Request for Assistance with GitHub Students Developer Pack Document Verification

### DISCUSSION THREAD

**Discussion by ghost** [community/community#81853] [GitHub Education]
### Hi everyone,

Hello GitHub Support Team,

I hope this message finds you well. I am reaching out to seek assistance regarding the verification process for the GitHub Students Developer Pack.

I am currently in the process of uploading the document to verify my student status, and I have encountered a challenge. The photo capture section allows for only low-quality images, making it difficult for me to provide a clear and satisfactory document.

I understand the importance of accurate verification and want to ensure that the document I submit meets the necessary standards. However, due to the limitations in the photo capture section, I am unable to provide a high-quality image.

Could you please guide me on how to proceed in such a situation? Is there an alternative method for document submission, or is there a workaround to upload a clearer image?

I appreciate your prompt attention to this matter and your assistance in ensuring a smooth verification process. Thank you for your time and support.

Best regards,

[Efe Yazgı]
[efeyazgii]
[efeyazgi@yahoo.com]

**Comment by ananuness** [comment]
Hii Efe! Others have faced issues in this step too and there is a workaround you can check out in this [discussion](https://github.com/orgs/community/discussions/81605). I hope it helps!!

**Comment by faridnec** [comment]
Hey in my case, it wasn't a problem. So before I submit with the direct camera, I tried to upload with pdf file. It didn't work they keep telling me to fix things but then I tried to taking pic of my student card and it worked. It wasn't that good quality too to be honest but they accepted. Maybe you can try it!

Hope this helps

**Comment by iSiddharth20** [comment]
What I did was, get all documents in a PDF File and then just use a PDF compression tool online. 
It reduced file size while maintaining all necessary information. 
Things looked clear to me even after significant file size reduction and student status was also verified.

Hope this works for you, Good Luck!
